reports/platopsapps/README.md

```diff
@@ -0,0 +1,60 @@
+# AKS Version Report
+
+This report is designed to check the deployed versions where applicable of Docmosis, Camunda and Flux.
```

Comment:
Maybe make the apps a list below this so it can be appended to in future?

```
This report is designed to check the deployed versions, where applicable, of:
- Docmosis
- Camunda 
- Flux

```